### DISCUSSION TITLE
Export doesnt work

### DISCUSSION THREAD

**Discussion by Fabian1410** [Budibase/budibase#2184] [General]
I want to export the internal BB Datasource, but the csv export only export json with this error :) 
I recently updated to the new version 0.9.86.

`{"message":"Missing/invalid DB name","status":500}`

**Comment by mjashanks** [comment]
Hey @Fabian1410 . This is indeed a bug. Thank you for reporting, I logged the issue here: https://github.com/Budibase/budibase/issues/2193
